pfcpiface/config.go

```diff
@@ -174,26 +174,27 @@ func validateConf(conf Conf) error {
 	return nil
 }
 
-// LoadConfigFile : parse json file and populate corresponding struct.
+// Remove comments from JSONC file
+func removeComments(jsonc string) string {
+	commentRegex := regexp.MustCompile(`(?m)//.*$|/\*.*?\*/`)
+	return commentRegex.ReplaceAllString(jsonc, "")
+}
+
+// LoadConfigFile : parse jsonc file and populate corresponding struct
 func LoadConfigFile(filepath string) (Conf, error) {
 	// Open up file.
-	jsonFile, err := os.Open(filepath)
+	jsoncFile, err := os.ReadFile(filepath)
 	if err != nil {
 		return Conf{}, err
 	}
-	defer jsonFile.Close()
 
-	// Read our file into memory.
-	byteValue, err := io.ReadAll(jsonFile)
-	if err != nil {
-		return Conf{}, err
-	}
+	jsonData := removeComments(string(jsoncFile))
```

Comment:
Do we still need this?

Reply:
Yes, we need these changes (as shown below):
```bash
               Error Trace:    config_test.go:92
|               Error:          Received unexpected error:
|                               invalid character '/' looking for beginning of value
|               Test:           TestLoadConfigFile/all_sample_configs_must_be_valid
|               Messages:       config ../conf/upf.jsonc is not valid
```

Alternatively, we can revert the changes (i.e., not to use the `removeComments` function) and try to use a package that reads/loads JSONC files such as https://github.com/kokizzu/json5b. What are your thoughts/preference on this?